create.sh

```diff
@@ -170,6 +171,25 @@ elif [ "$TEMPLATE_KEY" = "php" ]; then
     _composer="php composer.phar"
   fi
   install_command="$_composer install"
+
+#### PHP LARAVEL ####
+elif [ "$TEMPLATE_KEY" = "php-laravel" ]; then
+  check_for_command "php"
+
+  if type -p composer > /dev/null; then
+    _composer=composer
+  else
+    php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+    php composer-setup.php
+    php -r "unlink('composer-setup.php');"
+    _composer="php composer.phar"
+  fi
+
+  # Generate application key
+  php artisan key:generate
+  
+  install_command="$_composer install && php artisan key:generate"
```

Comment:
is this intended to run `php artisan key:generate` twice?

Reply:
That was leftover from my first implementation. Removed now (and added Leaf PHP)